### ISSUE TITLE
Site Not Found - WordPress Site Networks & Multidev

### ISSUE THREAD

**Issue by rachelwhitton** [pantheon-systems/documentation#2544]
Re: BUGS-1273
Case 83986
## Issue Description
`Error: Site not found.` returned by terminus env:cc command on Multidev only, with a config such as: `$domain = strtolower( stripslashes( $_SERVER['HTTP_HOST'] ) );`

## Suggested Resolution
Document edge case in WordPress Site Networks doc(s) and provide known workaround. 
Maybe on this page: [Managing WordPress Site Networks](https://pantheon.io/docs/managing-wordpress-site-networks/)

cc @joshkoenig @ari-gold

**Comment by joshkoenig** [comment]
Thanks for starting this issue. We should work to pull in the existing work done by @danielbachhuber and @alexdicianu in debugging this for some customers. :)

**Comment by danielbachhuber** [comment]
Here's some more context, from the email I sent the customer:

> Thanks for sharing the script. I saw the problem immediately after running it locally:
>
> Success: Made 2419 replacements. Please remember to flush your persistent object cache with `wp cache flush`.
 [notice] Command: account.daniel-t1 -- wp search-replace dev-account.example.com daniel-t1-account.pantheonsite.io [Exit: 0]
> 
> The immediate problem is this indeterminate state where you've run search-replace, so you'd expect you could flush cache against the new URL, but the cache still has the old URL, so you need to flush cache against the old URL before the new URL works.
>
> [...]
>
> I've created an issue to provide better messaging:
> 
> https://github.com/wp-cli/search-replace-command/issues/12
> 
> However, even with all of this, you had another problem: you were missing a default DOMAIN_CURRENT_SITE in your wp-config.php. Here's one that worked for me:
> 
> https://dl.dropboxusercontent.com/s/3e7nixv5lik7v7u/2017-06-06%20at%201.49%20PM.png
> 
> On multisite, WP-CLI uses `DOMAIN_CURRENT_SITE` to determine the default URL context. If no `--url=<url>` is provided and `DOMAIN_CURRENT_SITE` is missing, then `HTTP_HOST` ends up as an empty value which means WordPress doesn't know which site to resolve to.

tl;dr: WP-CLI requires one of: `DOMAIN_CURRENT_SITE` set in `wp-config.php`, or `--url=<url>` passed as an argument.

**Comment by joshkoenig** [comment]
So we need to update the docs to make sure people set up a default `DOMAIN_CURRENT_SITE` when they are setting up a WP Site Network? Probably we should give some guidance on how best to do that.

**Comment by danielbachhuber** [comment]
> So we need to update the docs to make sure people set up a default `DOMAIN_CURRENT_SITE` when they are setting up a WP Site Network?

Correct.

> Probably we should give some guidance on how best to do that.

This works as a generic catch-all:

```
define( 'DOMAIN_CURRENT_SITE', $_ENV['PANTHEON_ENVIRONMENT'] . '-' . $_ENV['PANTHEON_SITE_NAME'] . '.pantheonsite.io' );
```

**Comment by danielbachhuber** [comment]
@rachelwhitton Which reference document does CSE typically use when guiding a multisite setup?

**Comment by rachelwhitton** [comment]
@danielbachhuber Oh, looks like we have some content on this already in the docs: [Getting Started with WordPress Site Networks: Modify DOMAIN_CURRENT_SITE](https://pantheon.io/docs/create-wordpress-site-networks/#modify-domain_current_site) looks like the first snippet is for custom domains and the second is for platform domains. cc @adamedgmond

**Comment by ttrowell** [comment]
@rachelwhitton I'd still like to see this in a troubleshooting section that highlights the error message that the end user sees even if it just directs them back to the set up doc.
